vue-model-api/src/internal/ReactiveINodeJS.ts

```diff
@@ -238,4 +242,24 @@ export class ReactiveINodeJS implements INodeJS {
   triggerChangeInAllChildren() {
     this.trackAndTriggerForAllChildren?.trigger();
   }
+
+  createReferenceToPreventThisFromBeingGarbageCollected(
+    trackOrTriggerFunction: () => void,
+  ) {
+    // The Vue-objects must reference this wrapper,
+    // so that the wrapper is not garbage collected when one of the Vue-objects it manages is still in use.
+    //
+    // The wrapper may be removed from the cache (see `toReactiveINodeJS` and `Cache.ts`) after being garbage collected.
+    // If it has been removed from the cache and a change (see `ChangeJS`) is triggered
+    // then the change will not be handled
+    // because the wrapper will not be in the cache used in `handleChange`.
+    //
+    // The wrapper should only be garbage collected and be removed from the cache
+    // if it and any of the Vue-objects it manages are not used anywhere.
+    (
+      trackOrTriggerFunction as {
+        _referenceToManagingReactiveINodeJS?: ReactiveINodeJS;
+      }
+    )["_referenceToManagingReactiveINodeJS"] = this;
```

Comment:
I have no good idea of how to test this, as it depends on triggering garbage collection.
I have published the SNAPSHOT release `0.0.1-ff25b1d-SNAPSHOT` and tested the fix together with the affected customer.